pkg/crc/constants/constants.go

```diff
@@ -83,32 +84,29 @@ func GetAdminHelperURL() string {
 	return GetAdminHelperURLForOs(runtime.GOOS)
 }
 
-func defaultBundleForOs(preset crcpreset.Preset) map[string]string {
+func GetDefaultBundle(preset crcpreset.Preset) string {
+	var bundleName strings.Builder
+
+	bundleName.WriteString("crc")
+
 	switch preset {
 	case crcpreset.Podman:
-		return map[string]string{
-			"darwin":  fmt.Sprintf("crc_podman_vfkit_%s_%s.crcbundle", version.GetBundleVersion(preset), runtime.GOARCH),
-			"linux":   fmt.Sprintf("crc_podman_libvirt_%s_%s.crcbundle", version.GetBundleVersion(preset), runtime.GOARCH),
-			"windows": fmt.Sprintf("crc_podman_hyperv_%s_%s.crcbundle", version.GetBundleVersion(preset), runtime.GOARCH),
-		}
+		bundleName.WriteString("_podman")
 	case crcpreset.OKD:
-		return map[string]string{
-			"darwin":  fmt.Sprintf("crc_okd_vfkit_%s_%s.crcbundle", version.GetBundleVersion(preset), runtime.GOARCH),
-			"linux":   fmt.Sprintf("crc_okd_libvirt_%s_%s.crcbundle", version.GetBundleVersion(preset), runtime.GOARCH),
-			"windows": fmt.Sprintf("crc_okd_hyperv_%s_%s.crcbundle", version.GetBundleVersion(preset), runtime.GOARCH),
-		}
-	default:
-		return map[string]string{
-			"darwin":  fmt.Sprintf("crc_vfkit_%s_%s.crcbundle", version.GetBundleVersion(preset), runtime.GOARCH),
-			"linux":   fmt.Sprintf("crc_libvirt_%s_%s.crcbundle", version.GetBundleVersion(preset), runtime.GOARCH),
-			"windows": fmt.Sprintf("crc_hyperv_%s_%s.crcbundle", version.GetBundleVersion(preset), runtime.GOARCH),
-		}
+		bundleName.WriteString("_okd")
 	}
-}
 
-func GetDefaultBundle(preset crcpreset.Preset) string {
-	bundles := defaultBundleForOs(preset)
-	return bundles[runtime.GOOS]
+	switch runtime.GOOS {
+	case "darwin":
+		bundleName.WriteString("_vfkit")
+	case "linux":
+		bundleName.WriteString("_libvirt")
+	case "windows":
+		bundleName.WriteString("_hyperv")
+	}
+
+	fmt.Fprintf(&bundleName, "_%s_%s.crcbundle", version.GetBundleVersion(preset), runtime.GOARCH)
+	return bundleName.String()
```

Comment:
Nice 👍🏽

Reply:
This is really nice, I can cherry pick it 👍🏼